plugin/evm/vm_test.go

```diff
@@ -808,6 +815,137 @@ func TestIssueAtomicTxs(t *testing.T) {
 	assert.Equal(t, Accepted, status)
 	assert.Equal(t, uint64(2), height, "expected height of indexed export tx to be 2")
 	assert.Equal(t, indexedExportTx.ID(), exportTx.ID(), "expected ID of indexed import tx to match original txID")
+
+	// The same atomic transaction is included in multiple processing blocks.
+	// It gets dropped in one block and is still processing in the other.
+	// getAtomicTx should return status Processing
+	exportTx2, err := vm.newExportTx(vm.ctx.AVAXAssetID, 1, vm.ctx.XChainID, testShortIDAddrs[0], initialBaseFee, []*secp256k1.PrivateKey{testKeys[0]})
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Can we separate the flow from this point on into its own test? This test is already pretty big, and I'd prefer to separate the basic flow test from the regression test for this exact behavior.

Reply:
👍🏽